.github/workflows/manual.yaml

```diff
@@ -10,7 +10,14 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v2
-
+    - name: Configure NASA Earthdata Login
+      if: ${{ 'EARTHDATA_USER' in secrets }}
+      env:
+        EARTHDATA_USER: ${{ secrets.EARTHDATA_USER}}
+        EARTHDATA_PASS: ${{ secrets.EARTHDATA_PASS }}
+      run: |
+        echo "machine urs.earthdata.nasa.gov login $EARTHDATA_USER password $EARTHDATA_PASS" > ~/.netrc
+        chmod 0600 ~/.netrc
```

Comment:
I think we can't move this to a shared action that we call where needed because it uses secrets, right?

Reply:
Yes, we could.

See my note regards the `if` condition for the comment above.

Reply:
each secret would have to be defined as an input for the composite action:  https://github.com/ICESAT-2HackWeek/website2022/blob/b14d3e2f607bdc461d9e927fe5170a493c664372/.github/workflows/deploy.yaml#L37